runtime/src/tests/functionalTests/ui/planner/workitem-quickpreview.ts

```diff
@@ -91,6 +100,27 @@ export class WorkItemQuickPreview extends ui.BaseElement {
     await this.iterationSaveButton.click();
   }
 
+  private async addComment(comment: string) {
+    await this.loadingAnimation.untilAbsent();
+    await this.commentsField.clickWhenReady();
+    await this.commentsInputField.sendKeys(comment);
+  }
+
+  async addCommentAndSave(comment: string) {
+    await this.addComment(comment);
+    await this.commentSaveButton.click();
+  }
+
+  async addCommentAndCancel(comment: string) {
+    await this.addComment(comment);
+    await this.commentCancelButton.click();
```

Comment:
We can use `this.commentCancelButton.clickWhenReady()` also. This makes sure the button is ready and clickable